app/util/PlutoMessageConsumer.scala

```diff
@@ -0,0 +1,63 @@
+package util
+
+import data.UnpackedDataStores
+import akka.actor.Scheduler
+import scala.annotation.tailrec
+import scala.concurrent.duration._
+import play.api.libs.json._
+import com.amazonaws.services.sqs.model.{DeleteMessageRequest, ReceiveMessageRequest, Message}
+import scala.concurrent.ExecutionContext
+import scala.collection.JavaConversions._
+import com.gu.media.logging.Logging
+import data.DataStores
+
+case class PlutoMessageConsumer(val stores: DataStores, awsConfig: AWSConfig)
```

Comment:
Ah didn't realise it was SQS not Kinesis. Either way, since the call blocks we should consider running this in its own Akka dispatcher to avoid blocking the threads on the main app